fmriprep/cli/run.py

```diff
@@ -9,21 +9,34 @@
 
 import os
 import os.path as op
-import glob
+import logging
 import sys
 import uuid
+import warnings
 from argparse import ArgumentParser
 from argparse import RawTextHelpFormatter
 from multiprocessing import cpu_count
 from time import strftime
+from fmriprep.info import __version__
 
-from niworkflows.nipype import config as ncfg
+logging.addLevelName(25, 'INFO')  # Add a new level between INFO and WARNING
```

Comment:
We can call it IMPORTANT or so, but it is just the name that appears in the logs.

I made this because the default log level in python is WARNING, so those loggers not handled with nipype were writing very little information. With this extra level of detail we can have some INFO messages that are printed by default, and some INFO more verbose.

The alternative is give this level 25 a name like IMPORTANT. But I don't think there is any gain.

In the long term, we should aim to control all the logs (nipype included) in a centralized way. My impression is that INFO is too detailed in nipype, with this new level we will easily filter those out.

Also, an easier way to specify the verbosity would be nice. In MRIQC I started with `-v`, `-vv`, `-vvv`, etc. to increase the verbosity level and it works very well.